login/service_whitebox_test.go

```diff
@@ -156,8 +156,7 @@ func TestEncodeTokenOK(t *testing.T) {
 	refreshToken := "refreshToken%@!/\\&?"
 	tokenType := "tokenType%@!/\\&?"
 	expiresIn := 1800
-	var refreshExpiresIn float64
-	refreshExpiresIn = 2.59e6
+	refreshExpiresIn := 2.59e6
```

Comment:
@alexeykazakov Oh right, I did not think about platform specific code.
thank you, I will undo changes soon.